recipes/libhdbpp/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "libhdbpp" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/tango-controls/hdbpp/libhdbpp/-/archive/v{{ version }}/libhdbpp-v{{ version }}.tar.gz
+  sha256: 53e1647d0106787adaed5d9e82c3eb53c5d5b1a4892387b1e52a183fadde4b93
+
+build:
+  number: 0
+  skip: true  # [not linux]
```

Looks like upstream supports linux and windows.

```suggestion
  skip: true  # [osx]
```

---

pkg-config is available for Windows? CMake does not require platform specific directions. Does pkg-config require platform specific directions? I don't know; I've never used pkg-config.